ui/apos/components/AposImportModal.vue

```diff
@@ -29,15 +33,15 @@
           <AposLabel
             label="aposImportExport:importWarning"
             class="apos-import__warning"
-            :modifiers="[ 'apos-is-warning', 'apos-is-filled' ]"
+            :modifiers="['apos-is-warning', 'apos-is-filled']"
           />
           <div class="apos-import__import-drafts-only-wrapper">
             <AposCheckbox
               v-model="checked"
               class="apos-import__import-drafts-only"
               :choice="{
                 value: 'importDraftsOnly',
-                label: $t('aposImportExport:importDraftsOnly')
+                label: $t('aposImportExport:importDraftsOnly'),
```

Comment:
odd

Reply:
Right, probably prettier in action or IDK, removed.